editor/src/clj/editor/gui.clj

```diff
@@ -3011,18 +3011,20 @@
   (concat base-node-type-infos @custom-node-type-infos))
 
 (defn- get-registered-node-type-info [node-type custom-type]
-  (let [infos (filter (fn [info] (and
-                                  (= (:node-type info) node-type)
-                                  (= (:custom-type info) custom-type))) (get-registered-node-type-infos))
-        info (first infos)]
-    info))
+  (or (some (fn [info]
+              (when (and (= node-type (:node-type info))
+                         (= custom-type (:custom-type info)))
+                info))
+            (get-registered-node-type-infos))
+      (throw (ex-info (format "Unable to locate GUI node type info. Extension not loaded? (node-type=%s, custom-type=%s)"
```

Comment:
This error will be logged, but will cause the resource node to be marked defective, and trying to open it will show an error message in the style of "Unable to open 'spine.gui', since it contains unrecognizable data. Could the project be missing a required extension?".